src/index.js

```diff
@@ -1,3 +1,4 @@
+const os = require('os');
```

Comment:
the 'os' module does work in webpack. it's using: https://github.com/webpack/node-libs-browser, which in turn uses: https://github.com/CoderPuppy/os-browserify. that being said, I'm not so sure anymore if that is the right approach for src/index.js, since the new line escape sequence is always '\n', no matter which browser. if src/index.js is only meant to run from within webpack, it might as well be fine.